### ISSUE TITLE
acrylic effect goes away when window loses focus

### ISSUE THREAD

**Issue by ethanhowell** [microsoft/terminal#3497]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.18362.418]
Windows Terminal version (if applicable): 0.6.2951.0

Any other software?
```

# Steps to reproduce

1. Open up Windows Terminal. Open up any other window. Make sure windows terminal has the useAcrylic set to tru and the opacity set to something like .5. Snap the two windows to either side. When Terminal is in focus the acrylic will work. When it's not it will not.

# Expected behavior
The acrylic effect to stay whether or not terminal is in focus.

# Actual behavior
The acrylic effect only appears when terminal is in focus

**Comment by d-bingham** [comment]
This is just part of how the Windows UI works -- Acrylic effects fade in and out depending on focus.

**Comment by WorldMaker** [comment]
Terminal feels like a case where it would be great to have the opposite: acrylic to blend into the background when inactive and not when focused.

**Comment by zadjii-msft** [comment]
This has been discussed to death why it's not possible unfortunately.

> Unfortunately this is by design.
> The system-level policy for all acrylic applications is that only the active/focused window is allowed to display it.
> So there is no parameter or config that you can use to always display it.

> This is, unfortunately, by design. System policy dictates (and Windows controls this, not Terminal(!)) that only a currently-active window can be displayed with the acrylic effect.

> It may be difficult due to the limitations of the OS.
> 
> [docs.microsoft.com/en-us/windows/uwp/design/style/acrylic#usability-and-adaptability](https://docs.microsoft.com/en-us/windows/uwp/design/style/acrylic#usability-and-adaptability)
> 
> > In addition, only background acrylic will **replace its translucency and texture with a solid color**:
> > 
> > * When an app window on desktop deactivates
> > * When the UWP app is running on phone, Xbox, HoloLens or tablet mode


/dup #2326 #1886 #1747 #1546 #1505 #1099 #593 #3336 #2763 #2698

**Comment by Pitta** [comment]
Just realized I could do this. 
Came here to be sad about the out of focus lack of acrylic blur. 

This is a bummer.

**Comment by zadjii-msft** [comment]
@Pitta If you'd take a look at #7158, there's a potential solution that just requires a decent amount of work to investigate and apply.

**Comment by caywen** [comment]
Argh, this is a useless feature! The window in focus has way less contrast then the ones not in focus? It doesn't make sense - what's the thinking behind how this was implemented? Also, what's the point of the retro terminal effect? Feels like misplaced priorities.

**Comment by Pitta** [comment]
I wouldn't go that far. The app is open source and the internal maintainers are not likely to be core windows development that can make that a priority.

I'll dump on MS for totally breaking stuff on the mobile Xbox app and ignoring it, but this just seems unfortunate and ultimately not super important.

**Comment by soumalyatheking22012001** [comment]
I came here thinking it would just be a tweak to settings.json but I found it as a bummer...

**Comment by stefanba3** [comment]
> Just realized I could do this.
> Came here to be sad about the out of focus lack of acrylic blur.
> 
> This is a bummer.

Sadness

**Comment by rohilpinto** [comment]
:( terminal looks ugly when its fully black

**Comment by avfas** [comment]
> It may be difficult due to the limitations of the OS.

Good thing a new OS is here to save us; Windows 11. 

PS: Not going to happen

**Comment by 5kur4** [comment]
this sucks :(

**Comment by Ark2000** [comment]
stupid microsoft

**Comment by zadjii-msft** [comment]
# If you'd read, you'd find that #15923 was merged last month, fixing #7158 (and this issue).
